installing/installing_nutanix/installing-nutanix-three-node.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: ASSEMBLY
+[id="installing-nutanix-three-node"]
+= Installing a three-node cluster on Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: installing-nutanix-three-node
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a three-node cluster on Nutanix. A three-node cluster consists of three control plane machines, which also act as compute machines. This type of cluster provides a smaller, more resource efficient cluster, for cluster administrators and developers to use for testing, development, and production.
+
+include::modules/installation-three-node-cluster-cloud-provider.adoc[leveloffset=+1]
+
+== Next steps
```

Comment:
The next steps link might be confusing in that one has created a three-node cluster on Nutanix and then needs to create a generic cluster? Should the next steps addition be padded out to explain the next steps and then follow with a link or drop a user to the exact section in installing a cluster on Nutanix where they should continue?

Or maybe `Installing a three-node cluster on Nutanix` (renamed to configuring a three-node cluster environment on Nutanix) should be moved above `Installing a cluster on Nutanix`

Reply:
The user has, in fact, not created a three-node cluster at this point. The purpose of this topic is to detail only those steps necessary to create a three node cluster. The purpose of the placement of this content in the TOC is for findability. If I am a customer looking to install a three-node cluster, Stephanie and I did not want to bury the information inside of an assembly. I am comfortable leaving the content as is, as this is the approach we took in 4.13, when introducing 3-node support for Azure, AWS, GCP, and vSphere.

Reply:
My concern is that this isn't truly a procedure that does a full action. It is more of an example. But, with the same format in 5 places, horse and barn, I reckon